lib/Model/Listing/AbstractListing.php

```diff
@@ -44,9 +44,9 @@ abstract class AbstractListing extends AbstractModel implements \Iterator, \Coun
     protected $limit;
 
     /**
-     * @var int|null
+     * @var int
      */
-    protected $offset;
+    protected $offset = 0;
```

Comment:
@brusch does this fall under the backwards compatibility promise that we discussed earlier or not? This seems to be for pimcore 11, so I am a bit confused!

Reply:
We can introduce breaking changes since this is for Pimcore 11.